wrangler-transform/src/e2e-test/features/Wrangler/DataTypeParsers.feature

```diff
@@ -24,8 +24,6 @@ Feature:  datatype parsers
     Then Replace input plugin property: "project" with value: "projectId"
     Then Replace input plugin property: "dataset" with value: "dataset"
     Then Replace input plugin property: "table" with value: "bqSourceTable"
-    Then Click on the Get Schema button
-    Then Click on the Validate button
```

Comment:
why are these steps removed in all PRs?

Reply:
As the pipelines are exported.In almost all the PR's the issue is when we try to get schema then randomly in most of the fields it is going null when we click on get schema and validate button. And it also changes the schema . So just to remove that issue we removed that part from here.

Reply:
Resolved the issue and added both the steps.